src/components/withFormik.jsx

```diff
@@ -44,8 +44,8 @@ export default withFormik({
 		Telephone: '',
 		location: '',
 		describeTheProblem: '',
-		Latitude: '',
-		Longitude: '',
+		Latitude: '39.4001526',
```

Comment:
I'm not 100% sure on this, but I think the only change you might need for this is to set the lat and long to decimal here, then you won't need to parseFloat anything?

```js
Latitude: 39.4001526,
Longitude: -76.6074448,
```

Reply:
use decimal instead of string for Latitude and Longitude